### PR TITLE
test: increase workflow timeout

### PR DESCRIPTION
**Pull request by j-zimnowoda**
## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [ ] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [ ] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [ ] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

**Comment by Morriz** [comment]
Why is a workflow timeout needed if I may ask? To get out of an error loop/hangup I presume. We should then time it so that it finishes within a performance window that we choose. If it does not install within that window we know something is wrong with the perf of that cluster, and we should choose a better performing cluster perhaps?

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -149,7 +158,7 @@ jobs:
       - name: Create k8s cluster for testing
         run: |
           doctl kubernetes cluster create ${{ env.DIGITALOCEAN_CLUSTER_NAME }} \
-            --tag integration-test \
+            --tag source:github \
```
Comment (staticvoid255) [inline .github/workflows/integration.yml:161]: Maybe we can decide on a tag for temporary resources so we can run automated cleanup

**Review by staticvoid255 — APPROVED** [review]
Totally fine with this solution, but I'd be more on board with increasing the node size

File: .github/workflows/integration.yml (hunk 1 above)
Comment (j-zimnowoda) [reply .github/workflows/integration.yml:161]: Good point, I want to create a Github pipeline to remove all clusters with certain tags, so if someone forget removing adhoc cluster then it gets deleted at night anyway. TBD as a separate PR

**Pull request merged**